apps/app/src/pages/search/[...params]/index.tsx

```diff
@@ -43,6 +43,77 @@ const ServiceFilter = dynamic(() =>
 	import('@weareinreach/ui/modals/ServiceFilter').then((mod) => mod.ServiceFilter)
 )
 
+const stateRiskLevels = {
```

Comment:
Sorry this is a bit ugly, I could do 4 arrays and then build some logic if else statements, not sure.